### ISSUE TITLE
[Zone] Shrine of Goldrinn phasing

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Blackwing-Lair#1847]
Map: 1 (Kalimdor)
Zone: 616 (Mount Hyjal)
Area: 4979 (Shrine of Goldrinn)

This area should undergo phasing as you progress with the quest chains.
Currently the phasing is set wrong.

**How it should work:**
When you first arrive in the area with the quest leading you there (https://www.wowhead.com/quest=25269/the-voice-of-logosh or https://www.wowhead.com/quest=25268/the-voice-of-goldrinn respectively horde/ally) you should see a phase where the tents are burning, the wolves (https://www.wowhead.com/npc=39445/lycanthoth-vandal) have overrun the place and the quest giver NPCs to the left are imprisoned in cages. You should not see any friendly NPCs in the right side camp that burns.
According to the GPS this would be **phase 195.**

**The following NPCs must be visible in this phase:**
ALL https://www.wowhead.com/npc=39445/lycanthoth-vandal (wolves)
Specific NPCs: https://www.wowhead.com/npc=39432/takrik-ragehowl with DB GUID 372834
https://www.wowhead.com/npc=39435/royce-duskwhisper DB GUID 372836
https://www.wowhead.com/npc=39434/rio-duran DB GUID 372835
https://www.wowhead.com/npc=43547/mirala-fawnsinger DBGUID 372838
https://www.wowhead.com/npc=39433/ian-duran DB GUID 372852
Note: These NPCs all have a twin in the camp to the right which should activate in the next phase, and those npcs should not be visible in this phase yet. These npcs have the same name, same NPC id, only their DB GUID alters since they are in different position.

**The following NPCs must NOT be visible in this phase:**
ALL https://www.wowhead.com/npc=39637/goldrinn-defender
ALL https://www.wowhead.com/npc=39588/hyjal-stag
Specific NPCs: https://www.wowhead.com/npc=39432/takrik-ragehowl with DB GUID 372931
https://www.wowhead.com/npc=39435/royce-duskwhisper DB GUID 372933
https://www.wowhead.com/npc=39434/rio-duran DB GUID 372932
https://www.wowhead.com/npc=43547/mirala-fawnsinger DBGUID 372769
https://www.wowhead.com/npc=39433/ian-duran DB GUID 372936

Once you hand in the quest https://www.wowhead.com/quest=25273/lycanthoth-the-corruptor /https://www.wowhead.com/quest=25272/lycanthoth-the-corruptor ally/horde respectively, the phasing shifts.
In this new phase (which seems to be the default phase with no phase number, flags:8), the NPCs that were visible in the above described phase 195 should all disappear (wolves and left side camp GUIDs), and the NPCs which weren't visible (stags, defenders, right side camp GUIDs) should become visible.

**How it works right now:**
The phase you arrive in (phase 195), has wrong NPCs visible, you cannot see the wolves that you need for your quest objective, but you can see both camps of specific npcs, instead of just the left side (which looks pretty stupid).
You also cannot see the cages around those left side NPCs and the fires burning the tents in the right side camp, but that part is cosmetic.

When you hand in the quest Lycanthoth the corruptor, the phasing shifts as intended, gets you into the default phase.
However the default phase also shows wrong NPCs, the wolves (which you no longer need) now become visible, and the quest NPCs are still visible in both camps instead of just the new one (the right side).

**Comment by kDracku** [comment]
Confirm ! If you want to level from 80 to 83 you cant because there are so many zone phasing there

**Comment by giles1508** [comment]
something seems to have regressed with this,   i completed this absolutely fine the other day on my DK by following this quest order https://wow.gamepedia.com/Mount_Hyjal_quests   and now im doing the exact same thing on my mage,   the exact same way,  same order,   and im phased and i have to range pull them out of the phase,    ,  the only thing thats changed here is that when i was here last time,   all the npcs had quests on them,    but now the npcs are still there !!   but the quests have vanished,  and they are not in cages,  so its still in the wrong phase   this needs to be re looked at

**Comment by giles1508** [comment]
also to add,  the NPC called  Takrik Ragehowl    https://www.wowhead.com/npc=39432/takrik-ragehowl  is still giving   "  sweeping the shelf "   and  " into the Maw! "  quests   which is causing players extra confusion

**Comment by giles1508** [comment]
also when you complete the quest   https://wow.gamepedia.com/The_Shrine_Reclaimed    it puts you back into the phase with the npcs in the cages and they have all the quests available again to take in the wrong order,  without the prereqs...

**Comment by zippoking** [comment]
Yeah, this phasing issue is ridiculous. Just outside of Shrine of Goldrinn you can see the mobs, but once you are in this are, they vanish.
![Phasing Hyial - how it is 2](https://user-images.githubusercontent.com/18005338/59569563-aedbb900-908b-11e9-8f84-3fdd4ee30d1b.png)
![Phasing Hyial - how it is 1](https://user-images.githubusercontent.com/18005338/59569564-aedbb900-908b-11e9-8110-4721448bebd1.png)

**Comment by Dranah** [comment]
@giles1508 possibly it had something to do with you being a death knight, because the phasing never worked to anyone else, even if they followed quest order.

**Comment by zippoking** [comment]
I just tested the the quest chain at Shrine of Goldrinn with my shaman. And it is beautiful. All quests work, everything is doable and the phasing works for my Draenei Shaman. (see screenshot 1)

Why do i say that? Because on my night elf druid phasing does not work all all. (see screenshot 2).

Questchain on draenei shaman works perfectly:
![Shrine of Groldrinn Phasing - how it should be 1](https://user-images.githubusercontent.com/18005338/61036822-24f0e880-a3ca-11e9-9c86-5c5b1122e7e8.png)

Quest chain on night elf druid does not work at all:
![Shrine of Groldrinn Phasing - how it is 1](https://user-images.githubusercontent.com/18005338/61036824-24f0e880-a3ca-11e9-9238-bf7359624486.png)

**Comment by Rushor** [comment]
@zippoking could you check together with a gamemaster in which phase your shaman and in which phase you nightelf is? maybe you just got a wrong phasing aura somewhere

**Comment by Echinaceax** [comment]
Totally agree phasing is different on Classes, very helpful info guys thanks

**Comment by Echinaceax** [comment]
>could you check together with a gamemaster in which phase your shaman and in which phase you nightelf is? maybe you just got a wrong phasing aura somewhere

@Rushor - Its pretty obvious now why some have issues and some dont, some players delivered to the cage npcs (phase 1) with no issue, while others delivered to the none caged npc (phase two) = phase issues :)


### By the way anyone having phasing issues in Hyjal here is the workaround for it.

Me and Thinky started Hyjal last night and I noticed an issue with 2 sets of (?) for the quest return **Takrik Ragehowl**. when you are sent to  **Takrik Ragehowl** after all quests have been done up to being sent to  **Shrine of Goldrinn** deliver the first quest questback for him (open your map you will see 2 sets (?) IF you chose the wrong **Takrik Ragehowl** thats when your in the wrong phase. you need to chose the **Takrik Ragehowl** which is the one in the cage (which is phase one) then all the quests and phasing issues are gone (we did it so i know its working as i explained above) and i researched just now and got the proof for you all. the only issue in Hyjal we did encounter up to now was this part  (**Through the Dream**) still has a slight issue : workaround for that is to walk up to the dragon then stand at the tree on the right then the dragon is visable and you can deliver the quest.

(horde this ofc) but Ally same thing. the caged **CageIan Duran** npc not the other one

![kakrik](https://user-images.githubusercontent.com/39439201/77429597-33f1bc80-6dda-11ea-9863-8a425a3551e3.jpg)
 
![ian](https://user-images.githubusercontent.com/39439201/77431413-f5a9cc80-6ddc-11ea-8a7c-700828329e95.jpg)

![left side](https://user-images.githubusercontent.com/39439201/77429617-3bb16100-6dda-11ea-9e19-09720bcc0fa7.jpg)
